vk_api.py

```diff
@@ -0,0 +1,19 @@
+import requests
+from settings import TOKEN
+import time
+
+FRIENDS_GET_API_REQUEST = "https://api.vk.com/method/friends.get"
+
+
+class VkApi:
+    def getFriends(self, id):
+        time.sleep(0.4)
+        response = requests.get(FRIENDS_GET_API_REQUEST, params={'user_id': id,
+                                                                 'access_token': TOKEN,
+                                                                 'v': '5.52'
+                                                                 }).json()
+        try:
+            return response['response']
```

Comment:
Можно просто вот так, без try
```suggestion
            return response.get('response', [])
```